### ISSUE TITLE
Made for you page extension Shortcut is missing in all version of spotify

### ISSUE THREAD

**Issue by Jyrabun** [spicetify/cli#3194]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 
- Spicetify version:
```

### 📝 Description

Made for you shortcut in the marketplace does not work it does not even appear in older and more recent version of spotifty

**Comment by rxri** [comment]
how it's related to spicetify really? It's spotify's thing.

**Comment by Jyrabun** [comment]
Its related due to it being a part of spicetify marketplace. an addon that is downloaded from the same software

**Comment by AylenSK** [comment]
Same here.
I reported this issue on Spicetify-apps 3 days ago and got no answer :(
https://github.com/Pithaya/spicetify-apps/issues/50

**Comment by rxri** [comment]
> Its related due to it being a part of spicetify marketplace. an addon that is downloaded from the same software

Let me rephrase. How it's related to cli then? It's not. The snippet may just not work on new versions and that's all.
